subsys/bluetooth/mesh/lightness_srv.c

```diff
@@ -856,6 +858,18 @@ static int bt_mesh_lightness_srv_start(struct bt_mesh_model *mod)
 	lightness_srv_change_lvl(srv, NULL, &set, &dummy);
 	return 0;
```

Comment:
Is it really required to return value? Since it is an internal API probably it is worth simplifying the prototype.
As a recommendation, I do not insist.

Reply:
The return value is used in `bt_mesh_lightness_srv_start`.